raft.go

```diff
@@ -174,44 +189,43 @@ func (r *Raft) runCandidate() {
 	r.logger.Printf("[INFO] raft: %v entering Candidate state in term %v",
 		r, r.getCurrentTerm()+1)
 	metrics.IncrCounter([]string{"raft", "state", "candidate"}, 1)
+	// defer metrics.MeasureSince([]string{"raft", "candidate", "electSelf"}, time.Now())
```

Comment:
This'll have the time to complete an election from the candidate's perspective (one election cycle). Probably useful to just uncomment.